tactics/hints.ml

```diff
@@ -1730,8 +1730,9 @@ let pr_hint_db_env env sigma db =
       hov 0 (goal_descr ++ str " -> " ++ hints)
     in
     let hints =
+      let str h = Pp.string_of_ppcmds (pr_global h) in
```

Comment:
If the names are qualified, this will sort using the module name first. Don't we want instead to sort per the label of the global, which is more stable and probably the one we actually care about?

Reply:
Done.